docs/src/configuring_zed.md

```diff
@@ -142,6 +142,24 @@ For example, to disable ligatures for a given font you can add the following to
 
 `boolean` values
 
+## Centered Layout
+
+- Description: Configuration for the centered layout mode.
+- Setting: `centered_layout`
+- Default:
+
+```json
+"centered_layout": {
+  "left_padding": 0.2,
+  "right_padding": 0.2,
+}
+```
+
+**Options**
+
+The `left_padding` and `right_padding` options define the relative width of the
+left and right padding of the central pane from the workspace when the centered layout mode is activated. Valid values range is from `0` to `0.45`.
```

Comment:
Doc says max padding is 0.45 but Workspace::MAX_PADDING is defined as 0.4 above?

Reply:
Thank you for spotting, will update the docs according to the code, but if that was the wrong move and 0.45 should be used instead, future PRs can fix that.